package.json

```
@@ -11,7 +11,7 @@
         "lint:check": "npx eslint .",
         "test": "cross-env NODE_ENV=test npm run db:migrate && cross-env NODE_ENV=test npm run db:clean && cross-env NODE_ENV=test npm run db:seed && cross-env NODE_ENV=test vitest --config ./vitest.config.mjs",
         "dbml:generate": "ts-node src/config/db/utils/dbml.ts",
-        "db:generate": "drizzle-kit generate:mysql",
+        "db:generate": "drizzle-kit generate",
         "db:rollback": "drizzle-kit drop --config=drizzle.config.ts",
```

```suggestion
        "db:rollback": "drizzle-kit drop",
```
config maybe mention kora lagbe na ekhn